### PR TITLE
[prosiebensat1] Fixed Title Extraction. Issue #12318

### PR DESCRIPTION
**Pull request by ThomasChr**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

File: youtube_dl/extractor/prosiebensat1.py [hunk 1]
```diff
@@ -338,6 +338,7 @@ class ProSiebenSat1IE(ProSiebenSat1BaseIE):
         r'<header class="module_header">\s*<h2>([^<]+)</h2>\s*</header>',
         r'<h2 class="video-title" itemprop="name">\s*(.+?)</h2>',
         r'<div[^>]+id="veeseoTitle"[^>]*>(.+?)</div>',
+        r'<meta.*title.*content\s*=\s*"([^"]*)">',
```
Comment (dstftw) [inline youtube_dl/extractor/prosiebensat1.py]: This may match across several `meta`s.

**Comment by dstftw** [comment]
1. Write [conventional commit messages](https://chris.beams.io/posts/git-commit/).
2. No, you did not fix title extraction, you've just added another pattern that is what your should write in commit message.
3. Add a test.

**Comment by ThomasChr** [comment]
@dstftw Everything allright with my new commit?

**Comment by dstftw** [comment]
No, see the review comment.

**Comment by ThomasChr** [comment]
@dstftw I've changed the regex to be more specific in my last commit 6 days ago. Still not exact enough?

**Comment by dstftw** [comment]
Nothing really changed. `.` matches any character.

**Comment by ThomasChr** [comment]
Thats what you mean... I think [^>] should do the trick. If I'm right about that I will change the commit tomorrow.

**Comment by dstftw** [comment]
Proper way is to use `self._og_search_title`.

**Comment by ThomasChr** [comment]
I've simply added self._og_search_title called when the regex-extraction of the title failes.
I've removed the testcase because the Testvideo is gone now, so I think it makes no sense to leave the testcase in.

**Pull request merged**